core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -1179,12 +1179,12 @@ public CommitId flush(boolean force, boolean waitIfOngoing) throws EngineExcepti
                     try {
                         translog.prepareCommit();
                         logger.trace("starting commit for flush; commitTranslog=true");
-                        commitIndexWriter(indexWriter, translog, null);
+                        final long committedGeneration = commitIndexWriter(indexWriter, translog, null);
```

Comment:
I would like to focus on the naming in a small follow-up PR immediately after this PR.